adafruit_is31fl3731.py

```diff
@@ -136,19 +193,27 @@ def fade(self, fade_in=None, fade_out=None, pause=0):
         self._register(_CONFIG_BANK, _BREATH2_REGISTER, 1 << 4 | pause)
 
     def frame(self, frame=None, show=True):
+        """Set the current frame
+            :param frame: frame number; 0-7 or None. If None function returns current frame
+            :param show: True to show the frame; False to don't force a show
+        """
         if frame is None:
             return self._frame
         if not 0 <= frame <= 8:
             raise ValueError("Frame out of range")
         self._frame = frame
         if show:
-            self._register(_CONFIG_BANK, _FRAME_REGISTER, frame);
+            self._register(_CONFIG_BANK, _FRAME_REGISTER, frame)
```

Comment:
Add `return None` here at the end to get rid of the "inconsistent return statement" warning.

Reply:
ok